lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1167,6 +1167,10 @@ def __init__(
         *,
         port: int = 3100,
         rules_dir="/loki/rules",
+        endpoint_schema: str = "http",
+        endpoint_address: str = "",
+        endpoint_port: Union[str, int] = 3100,
```

Comment:
Isn't this already the value for `port`?

Reply:
Yes, it's the value for `_port` ... may be it is redundant.

Reply:
Then we need to change the naming scheme (and potentially break people) OR get rid of the `endpoint_` prefix and remove the duplicate.